ambry-api/src/main/java/com.github.ambry/commons/Threshold.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import com.github.ambry.utils.Pair;
+import java.util.Map;
+
+
+/**
+ * A data structure holding various thresholds based on different criteria.
+ */
+public class Threshold {
```

Comment:
I think calling this class Thresholds instead of Threshold will be good to reflect that there are multiple checks done.

Reply:
good point, refactored the class name in multiple files.